### ISSUE TITLE
Constructor should be public

### ISSUE THREAD

**Issue by kykungz** [OOP2018/guessing-game-pknn1#3]
PakanonGame's constructor accessibility level should be `public`, so that it can be called across packages.
https://github.com/OOP2018/guessing-game-pknn1/blob/d6e9712fb947ec49b23919b18be29c2b7371f44e/src/PakanonGame.java#L35-L37
